source/UniverseObjects.h

```diff
@@ -135,6 +135,7 @@ class UniverseObjects {
 	std::map<const Sprite *, std::string> landingMessages;
 	std::map<const Sprite *, double> solarPower;
 	std::map<const Sprite *, double> solarWind;
+	std::map<const Sprite *, const Sprite *> starIcon;
```

Comment:
might almost be worth a sort of starInfo class or struct, not for this PR though